src/backend/api/controllers/root.ts

```diff
@@ -119,9 +119,9 @@ export async function handleDecryptWithPrivateKeys(
       returnAsymmetricOptions,
     } = req.body
 
-    if (symmetricEncryptedPrivateKeys) {
-      await unwrapTransportEncryptedPasswordInSymOptions(symmetricEncryptedPrivateKeys, context)
-    }
+    // if (symmetricEncryptedPrivateKeys) {
```

Comment:
@traylewin symmetricEncryptedPrivateKeys is an array. We cannot check if it has a password attached to it. Commenting this out works fine. 
I guess we dont need this piece of code